semantic-conventions/src/tests/semconv/model/test_error_detection.py

```diff
@@ -450,6 +450,15 @@ def test_nameless_event(self):
         self.assertIn("at least one of name or prefix", msg)
         self.assertEqual(e.line, 2)
 
+    def test_condition_missing_contionally_required_attribute(self):
+        with self.assertRaises(ValidationError) as ex:
+            self.open_yaml("yaml/errors/wrong_conditionally_required_no_condition.yaml")
+            self.fail()
```

Comment:
Do not use `self.fail()`, here the assertRaises will fail already if there is no ValidationError raised.